pkg/adaptor/proxy/service.go

```diff
@@ -5,20 +5,20 @@ package proxy
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
 	"net"
-	"os/exec"
 	"strings"
 	"time"
 
+	"github.com/containerd/containerd"
 	"github.com/containerd/ttrpc"
 	"github.com/gogo/protobuf/types"
 	"github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/agent/protocols"
 	pb "github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/agent/protocols/grpc"
 
 	cri "github.com/containerd/containerd/pkg/cri/annotations"
 	crio "github.com/containers/podman/v4/pkg/annotations"
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
```

Comment:
How about `criapi` instead of `runtime`?

Reply:
Sure, it's OK to rename.

Reply:
Fixed